<launch>
  <node pkg="detector" type="plan.py" name="detect_node" output="screen"/>
</launch>